update.bat

```diff
@@ -23,7 +23,7 @@ set PYTHON="%VENV_DIR%\Scripts\python.exe"
 
 :install_dependencies
 echo installing dependencies
-%PYTHON% -m pip install -r requirements.txt --force-reinstall
+%PYTHON% -m pip install -r requirements-global.txt -r requirements-cuda.txt --force-reinstall
```

Comment:
as discussed, please add a requirements.txt again that combines -global and -cuda

Reply:
Added in https://github.com/Nerogar/OneTrainer/pull/193/commits/ef26ec3ae7a49fad8fa2443f109cec823677462c.